tiny/src/main.rs

```diff
@@ -125,12 +120,30 @@ fn run(
                     None
                 }
                 Ok(logger) => {
+                    // set logger configs
```

Comment:
This is the first step. It builds the configuration of the logger with the config file values